modules/server/src/main/scala/GrpcServer.scala

```diff
@@ -17,34 +17,34 @@
 package freestyle.rpc
 package server
 
-import freestyle.free._
+import freestyle.tagless.tagless
 import io.grpc._
 
 import scala.concurrent.duration.TimeUnit
 
-@free
-trait GrpcServer {
+@tagless
+trait GrpcServer[F[_]] {
```

Comment:
Why do you want to include the `F`? You can still use the `FS`.

Reply:
Yes, I know. IntelliJ shows less red code in this way.